pages/common/ls.md

```diff
@@ -35,3 +35,11 @@
 - Reverse the order of the results
 
 `ls -r`
+
+- List folders only without their contents in the current folder
```

Comment:
How about "List folders only, without their contents"? (with a comma and without the end of the sentence which is unnecessary IMO)